### ISSUE TITLE
Uploading empty sample names produce errors containing "NaN"

### ISSUE THREAD

**Issue by bede** [GlobalPathogenAnalysisService/gpas-cli#93]
https://oc-collab.gc3.ocs.oraclecloud.com/browse/C900000008-958

**Comment by bede** [comment]
Fixed 2cccd40ed61064dbb7a17b66b7bd8f8c4bed778e

**Comment by bede** [comment]
```
(gpas-cli-dev) bede@zizzle gpas-cli % gpas validate /Users/bede/Downloads/test_1\ 2.csv 
Traceback (most recent call last):
  File "/Users/bede/miniconda3/envs/gpas-cli-dev/bin/gpas", line 8, in <module>
    sys.exit(main())
  File "/Users/bede/Research/Git/gpas-cli/src/gpas/cli.py", line 265, in main
    defopt.run(
  File "/Users/bede/miniconda3/envs/gpas-cli-dev/lib/python3.10/site-packages/defopt.py", line 356, in run
    return call()
  File "/Users/bede/Research/Git/gpas-cli/src/gpas/cli.py", line 60, in validate_wrapper
    jsonify_exceptions(
  File "/Users/bede/Research/Git/gpas-cli/src/gpas/misc.py", line 129, in jsonify_exceptions
    return function(**kwargs)
  File "/Users/bede/Research/Git/gpas-cli/src/gpas/cli.py", line 39, in validate
    df, schema_name = validation.validate(upload_csv, allowed_tags)
  File "/Users/bede/Research/Git/gpas-cli/src/gpas/validation.py", line 527, in validate
    raise ValidationError(parse_validation_errors(e)) from None
gpas.validation.ValidationError: Failed to validate upload CSV (3 errors):
sample_name error                      
            sample_name cannot be empty
(gpas-cli-dev) bede@zizzle gpas-cli % 
```
